src/main/java/io/asyncer/r2dbc/mysql/MySqlBatchingBatch.java

```diff
@@ -34,7 +36,7 @@ final class MySqlBatchingBatch extends MySqlBatch {
 
     private final ConnectionContext context;
 
-    private StringBuilder builder;
+    private final StringJoiner queries = new StringJoiner(";");
```

Comment:
i think `StringJoiner` is sufficient and simple enough in this use case.
so i used it instead of `List`